pages/dst/index.md

```diff
@@ -11,6 +11,7 @@ display_title: Veterans Community Care Program Eligibility Tool
 
 # This line indicates that this page is not to be built to production (www.va.gov)
 vagovprod: false
+private: true
```

Comment:
Is this because it showing up in the Staging sitemap was causing issue?

Reply:
Yeah. I don't remember the exact reason. I think it came through @1Copenut?